src/Blocks/components/admin-listing/admin-listing.php

```diff
@@ -210,7 +188,7 @@
 if ($adminListingPageTitle || $adminListingSubTitle) {
 	$topBar = [
 		Components::render('layout', [
-			'layoutType' => 'first-left-others-right',
+			'layoutType' => $adminListingType !== 'trash' ? 'first-three-left-others-right' : 'first-left-others-right',
```

Comment:
May be wise to have the `trash` in a variable since it is being used extensively thorough the file 😊

Reply:
agreed